src/engineering-meeting/sketch/sketch.tsx

```diff
@@ -31,7 +31,7 @@ const sketch: Sketch = (sketch: P5Instance) => {
          sketch.resizeCanvas(canvasSize.width, canvasSize.height)
 
          const data = props.data as BuildingBlock[]
-         structure = StructureViewModel.from(data, sketch, canvasSize.height)
+         structure = StructureViewModel.from([...data].reverse(), sketch, canvasSize.height)
```

Comment:
`reverse()` is a mutating operation, we need to preserve `data` on it's original order so that when we paint the table representing the meeting structure it matches the order of what has been painted